oaknational/Oak-Web-Application
# feat: switch to oak-curriculum-schema

## Description

This change completes the work of [this rfc](https://www.notion.so/oaknationalacademy/Request-for-Comments-Npm-Module-for-a-Shared-Schema-between-Front-and-Backend-9cb199c3f0eb4ddab5e4c29785b7e0d5?pvs=4) to provide a shared set of contact tests between OWA and DBT. By deriving fixtures, schemas and types from this package we can improve confidence in the integrity of our data and how we handle it.

- List of changes

switched pupilLesson query schemas over to @oaknational/oak-curriculum-schema
switched fixtures where relevant 

## How to test

1. Go to https://deploy-preview-2376--oak-web-application.netlify.thenational.academy/pupils/lessons/transverse-waves
2. the lesson should load without issue
